src/situations/commun/vues/question_entete.vue

```diff
@@ -97,9 +101,15 @@ export default {
 
     demarreSon () {
       if (!this.question.intitule && this.afficheLectureQuestionAudio) {
-        this.demarreSonBouton('boutonLectureQuestionAudio', () => {
-          this.demarreSonBouton('boutonLectureTexteEntete');
-        });
+        if (this.demarreModaliteReponseEnPremier && this.afficheLectureTexteEntete) {
+          this.demarreSonBouton('boutonLectureTexteEntete', () => {
+            this.demarreSonBouton('boutonLectureQuestionAudio');
+          });
+        } else {
+          this.demarreSonBouton('boutonLectureQuestionAudio', () => {
+            this.demarreSonBouton('boutonLectureTexteEntete');
+          });
+        }
```

Comment:
À la lecture de ce code, j'ai l'impression que ça ne marche pas. Si afficheLectureTexteEntete est faux alors on passe dans le `else` qui va finir par démarrer le son du boutonLectureTexteEntete…

Reply:
Dans le code original, que `this.afficheLectureTexteEntete` soit true ou false on passait forcément dans la condition. Le test suivant me fait penser que c'est donc géré :

```
describe('#demarreSon', function (){
    it("démarre le son quand il y a une question audio uniquement", function () {
      const question = {
        retranscription_audio: 'audio de la réponse',
        reponses: [{ nom_technique: 'reponse1', type_choix: 'bon'}]
      };
      depotRessources.existeMessageAudio =
        (nom_technique) => nom_technique == 'reponse1';
      const vue = composant(question);
      vue.vm.demarreSon();
      expect(mockDemarreSonQuestionAudio).toHaveBeenCalled();
      expect(mockDemarreSonTexteEntete).not.toHaveBeenCalled();
      callbackFinQuestionAudio();
      expect(mockDemarreSonTexteEntete).not.toHaveBeenCalled();
    });
    });
```

Reply:
Ah ok, je comprends. Le code est quand même vraiment pas terrible avec des if de if, et un mélange de conditions logique. J'ai fait une PR simplificatrice. #1447